docker-dev/docker-compose.yml

```diff
@@ -0,0 +1,203 @@
+version: '2'
+
+services:
+
+### OMS TUNNEL        #######################################
+### Proxy container #######################################
+
+    traefik:
+        image: aegee/traefik:dev
+        ports:
+            - "80:80"
+            - "8080:8080"
+            - "443:443"
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock
+
+### service registry #######################################
+
+    omsserviceregistry:
+        image: aegee/omsserviceregistry:dev
+        ports:
+            - "7000:7000"
+        volumes:
+            - ./docker-compose.yml:/usr/src/docker-compose.yml:ro
+            - ../oms-serviceregistry:/usr/src/app
+            - shared:/usr/src/shared
+
+### OMS CORE        #######################################
+### Applications Code Container #############################
+
+    omscore-data:
+        image: tianon/true
+        volumes:
+            - "../oms-core:/var/www"
+            - shared:/var/shared
+
+### Workspace Utilities Container ###########################
+
+    omscore:
+        image: aegee/omscore:dev
+        volumes_from:
```

Comment:
`volumes_from` [will be deprecated soon ](https://docs.docker.com/compose/compose-file/compose-versioning/), so it's advisable (yet, silly..) to copy and paste (that's why I repeated them)